### PR TITLE
DM-47256: Remove apply_schema_to_tables flag

### PR DESCRIPTION
**Pull request by JeremyMcCormick**
This flag is redundant, because enabling `apply_schema_to_metadata` on the `MetaDataBuilder` will already result in the tables being associated to the schema correctly.

## Checklist

- [x] Ran Jenkins
- [x] Added a release note for user-visible changes to `docs/changes`

### CODE REVIEW

File: docs/changes/DM-47256.removal
Comment (andy-slac) [inline docs/changes/DM-47256.removal:1]: Does this file need an extension? Looking at the contents I think this should be restructured text, but I'm not sure what towncrier assumptions are?

**Review by andy-slac — APPROVED** [review]
Looks good, one minor question.

File: docs/changes/DM-47256.removal
Comment (timj) [reply docs/changes/DM-47256.removal:1]: It will need the .rst extension.

Comment (JeremyMcCormick) [reply docs/changes/DM-47256.removal:1]: Good catch - fixed now.

**Pull request merged**